lib/home/library/video_library.dart

```diff
@@ -118,110 +110,43 @@ class _VideoLibraryState extends State<VideoLibrary> {
   Future<void> fetchDocuments() async {
     final QuerySnapshot snapshot = await Firestore.streams.get();
     setState(() {
-      documents = snapshot.docs;
-    });
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    if (documents.isEmpty) {
-      fetchDocuments();
```

Comment:
Since `fetchDocuments()` was called in `initState` we can remove the second call here.